authentication/account/service/user_blackbox_test.go

```diff
@@ -66,28 +66,51 @@ func (s *userServiceBlackboxTestSuite) TestDeprovision() {
 }
 
 func (s *userServiceBlackboxTestSuite) TestDeactivate() {
+
 	s.T().Run("ok", func(t *testing.T) {
-		// given
+		// given 2 users with tokens
 		userToDeactivate := s.Graph.CreateUser()
+		token1 := s.Graph.CreateToken(userToDeactivate)
+		token2 := s.Graph.CreateToken(userToDeactivate)
 		userToStayIntact := s.Graph.CreateUser()
+		token3 := s.Graph.CreateToken(userToStayIntact)
+		token4 := s.Graph.CreateToken(userToStayIntact)
 		// when
 		identity, err := s.Application.UserService().DeactivateUser(s.Ctx, userToDeactivate.Identity().Username)
 		// then
 		require.NoError(t, err)
-		assert.False(t, identity.User.Active)        // user is inactive
-		assert.False(t, identity.User.Deprovisioned) // but user NOT deprovisionned
+		assert.False(t, identity.User.Active)        // user is inactive...
+		assert.False(t, identity.User.Deprovisioned) // ... but user NOT deprovisionned
 		assert.Equal(t, userToDeactivate.User().ID, identity.User.ID)
 		assert.Equal(t, userToDeactivate.IdentityID(), identity.ID)
-
+		// verify that user's fields were obfuscated
 		loadedUser := s.Graph.LoadUser(userToDeactivate.IdentityID())
 		require.NotNil(t, loadedUser)
 		testsupport.AssertIdentityObfuscated(t, userToDeactivate.Identity(), loadedUser.Identity())
-
+		// also, verify that user's tokens were revoked
+		for _, tID := range []uuid.UUID{token1.TokenID(), token2.TokenID()} {
+			tok := s.Graph.LoadToken(tID)
+			require.NotNil(t, tok)
+			assert.Equal(t, tok.Token().Status, token.TOKEN_STATUS_REVOKED)
+		}
+		// lastly, verify that user to keep intact remainded intact
 		loadedUser = s.Graph.LoadUser(userToStayIntact.IdentityID())
 		assert.True(t, loadedUser.User().Active)
 		testsupport.AssertIdentityEqual(t, userToStayIntact.Identity(), loadedUser.Identity())
+		for _, tID := range []uuid.UUID{token3.TokenID(), token4.TokenID()} {
+			tok := s.Graph.LoadToken(tID)
+			require.NotNil(t, tok)
+			assert.Equal(t, tok.Token().Status, token.TOKEN_STATUS_VALID)
```

Comment:
You can do this easier:

`assert.True(t, tok.Token().Valid())`

Reply:
ah cool, thanks @sbryzak !
change applied in be07719